x-pack/plugins/enterprise_search/public/applications/shared/layout/nav.tsx

```diff
@@ -204,9 +210,14 @@ export const useEnterpriseSearchEngineNav = (engineName?: string, isEmptyState?:
             },
             {
               id: 'enterpriseSearchApplicationsContent',
-              name: i18n.translate('xpack.enterpriseSearch.nav.engine.contentTitle', {
-                defaultMessage: 'Content',
-              }),
+              name: (
+                <EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
+                  {i18n.translate('xpack.enterpriseSearch.nav.engine.contentTitle', {
+                    defaultMessage: 'Content',
+                  })}
+                  {hasSchemaConflicts && <EuiIcon type="warning" color="danger" />}
```

Comment:
FYI for @julianrosado I don't think we will be able to reproduce this behavior when we add Search Applications to the serverless side nav. For serverless we will not have access to the application state so we cant fetch data etc.

Reply:
This is for the side nav if I'm following? That's fine as long as we surface it on the actual Search App landing page somehow.

Reply:
yea this is just the side nav. We can do it in dedicated b/c the side nav is rendered by the page. But for serverless the side nav is rendered by another plugin.